src/index.d.ts

```diff
@@ -1,8 +1,8 @@
 import * as React from "react";
-import PropTypes from "prop-types";
```

Comment:
is this file required?

Reply:
No, this can be ignored too as we have the typescript exports now @sshivananda

Reply:
yeah - I think we should delete this file 
there is an autogenerated dist/src/index.d.ts - that one should contain everything that is contained in this file

Reply:
Should I delete it with a commit? @sshivananda

Reply:
sure !

Reply:
Only one doubt, what about the comments of the props, do you want me to add those comments in the individual files where the type is defined for the component or we can ignore them now? @sshivananda

Reply:
we can add them in the individual files

Reply:
Okay